### ISSUE TITLE
Hangs on builtin-infoPlistUtility for NVMEFix

### ISSUE THREAD

**Issue by themelodian** [corpnewt/Lilu-and-Friends#39]
Script hangs on the following:
```
builtin-infoPlistUtility /var/folders/nw/jfv1pn153wg6xgj7ngznl_4w0000gn/T/tmpql2ekfam/NVMeFix/NVMeFix/Info.plist -producttype com.apple.product-type.kernel-extension -expandbuildsettings -platform macosx -o /private/var/folders/nw/jfv1pn153wg6xgj7ngznl_4w0000gn/T/tmpql2ekfam/NVMeFix/build/Release/NVMeFix.kext/Contents/Info.plist
```
when building NVMEFix.kext on MacOS 12.3.

**Comment by corpnewt** [comment]
Are you using Xcode 13.3?  I had issues building Lilu (and all other kexts I tried) with the latest Xcode on macOS 12.3, but Xcode 13.2.1 still works fine for me.  I would imagine it's something the Acidanthera team (and other affected kext authors) would have to adjust for - but if there's a local workaround that could be applied in the meantime, I'm open to input on how to fix it.

-CorpNewt

**Comment by themelodian** [comment]
I am using Xcode 13.3, yes.

**Comment by corpnewt** [comment]
Yeah - I'm betting that's the issue then.  That's unfortunately out of the scope of L&F unless there's some quick patch I could implement.

You could roll back to Xcode 13.2.1 and see if that works in the meantime.

-CorpNewt
